GMPWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -55,6 +56,64 @@ def getProcessors(tree):
   return processors
 
 
+# Replace with constants if pattern found in value(s)
+def replaceConstants(value, constants):
+  formatted_array = []
+  split_values = value.split()
+  for val in split_values:
+    captured_patterns = re.findall('\$\{\w*\}', val)
+    if len(captured_patterns) == 0:
+      formatted_array.append("\"{}\"".format(val))
+    elif len(captured_patterns) >= 1:
```

Comment:
```suggestion
    elif captured_patterns:
```
although `else` would also just work